### ISSUE TITLE
v2 breaks SSR apps

### ISSUE THREAD

**Issue by csorfab** [Stanko/animated-scroll-to#34]
The new 2.0.+ versions reference window when requiring the package. This breaks SSR apps, as window is not defined when running in a node environment. In 1.2.2 the problem was not present. It is very easy to reproduce.
```
mkdir anim-ssr-bug
cd anim-ssr-bug
npm install animated-scroll-to
node
> const a = require("animated-scroll-to")
```

result: 
```
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/csorfab/Web/test/node_modules/animated-scroll-to/lib/animated-scroll-to.js:95:22)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
...
```

**Comment by csorfab** [comment]
Submitted PR #35 that fixes this issue.

**Comment by Stanko** [comment]
I'm on it, thank you for the PR!

**Comment by Stanko** [comment]
Fixed in 2.0.3
https://github.com/Stanko/animated-scroll-to/commit/6423965e3a53886837f9b17283760684f7996f00

**Comment by csorfab** [comment]
Thanks for the quick action! :)
